.devops/pay-wallet-code-review-pipelines.yml

```diff
@@ -61,3 +61,77 @@ stages:
               summaryFileLocation: 'build/reports/jacoco/test/jacocoTestReport.xml'
               reportDirectory: 'build/reports/jacoco/test/html'
             displayName: 'Publish Code Coverage on Azure Devops'
+
+  - stage: IntegrationTest
+    dependsOn: []
+    jobs:
+      - job: Integration_Test
+        steps:
+          - script: cp env.example .env
+            displayName: 'Set env'
+          - script: docker compose up -d
+            displayName: 'Run app with docker-compose'
+
+          - script: |
+              timeout 180 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' localhost:8080/actuator/health/liveness)" != "200" ]]; do sleep 5; done' || exit 1
+            displayName: 'Health check pagopa-wallet-eventdispatcher'
+          - script: |
+              az storage message put \
+                --connection-string "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://localhost:10001/devstoreaccount1" \
+                --queue-name pagopa-wallet-cdc-queue \
+                --content '{
+                  "data":{
+                    "_id":"d283cbc5-cc48-4bc4-8f00-ddba4e24fc91",
+                    "walletId":"a527e843-9d1c-4531-ae5b-3809cc7abe7a",
```

Comment:
can you put this json in a file in the repo? so that it updates the file and automatically updates the test assertion?

Reply:
sure
done here b3770f66fc757dcca80399d81f5989835656a8cf